docs/datamodel/index.rst

```diff
@@ -4,7 +4,13 @@
 Data Model
 ==========
 
-EdgeDB is a relational database with strongly typed schema.
+On this page you'll find an overview of the fundamental types, objects, and concepts for EdgeDB, a relational database with strongly typed schema. Here's a quick overview of the links below:
+
+Types in EdgeDB include your own **Object Types** (e.g. *User*) and **Abstract Types** for other types to extend (e.g. *HasEmailAddress* for *User* and others can inherit), plus **Scalar Types** with single values (str, int64, etc.) and **Collection Types** like **arrays** and **tuples** for multiple values.
```

Comment:
Perhaps we should hyperlink `**Object Types**`, `**Abstract Types**`, and all other terms to their respective pages. I can quickly do that myself if you don't feel comfortable with ReST.

Reply:
Ah, it's ReST? I tried using just Markdown and it wasn't doing any hyperlinking so resorted to bolding in the meantime. Thanks!